bin/init_linter_git_hooks

```diff
@@ -125,5 +130,30 @@ def main():
   print("Success, githooks initialized!")
 
 
+def remove_linter():
```

Comment:
How will this function behave if (accidentally) invoked from submodule folder?

Reply:
It will replace the `pre-commit` hook for the submodule because we now support linter installation in submodules. :) The hooks of the parent repository aren't affected.